src/main/ruby/core/posix.rb

```diff
@@ -37,8 +37,8 @@ def self.to_nfi_type(type)
   def self.attach_function(method_name, native_name = method_name, argument_types, return_type)
     begin
       func = LIBC[native_name]
-    rescue RubyTruffleError => e
-      raise e unless e.message.include?('Unknown identifier')
+    rescue NameError # Missing function
+      func = nil
```

Comment:
Ah you should have said is what you wanted to solve!

Reply:
Yes :)